dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 dash_core_components==0.33.0
-dash_html_components==0.11.0rc5
-dash==0.32.0
+dash_html_components==0.13.4
+dash==0.32.2
```

Comment:
should we lock on to rc releases (github branch) here so we are testing against the latest of everything?

Reply:
We're already planning on making these point to head of master / release-v1 soon